### ISSUE TITLE
SELECT component: display issue in IE11

### ISSUE THREAD

**Issue by cheresier** [marcosmoura/vue-material#34]
When using the SELECT component in IE11, the scrollbars do not appear, and items spill outside the dropdown instead. I see this behavior both in my test page and in the official demo (https://marcosmoura.github.io/vue-material/#/components/select). See attachment for a screenshot.
![2016-10-26 15_47_50-vue material - internet explorer](https://cloud.githubusercontent.com/assets/18665877/19742575/a49aeda2-9b93-11e6-9d7c-70f4de975874.png)

**Comment by marcosmoura** [comment]
Oh. Yeah, this is definitely a bug. A sad one, right?

The select component it's being rebuilt using ```md-menu```. This new menu component is already finished. I will start to rework select on top of this menu pretty soon. Both will be released in the next minor version (v0.3.0).

I'm sorry to make you wait for that. Meanwhile I will find a workaround for you. I will post it here.
Thank you for reporting this!

**Comment by cheresier** [comment]
Thank you for being so responsive, and, most importantly, thank you for building this library and giving us a way to make our apps 1000% better-looking with little effort!

**Comment by marcosmoura** [comment]
@cheresier I have found the problem. IE 11 have too much trouble to render flexbox. On Edge those problems were fixed.
For now the workaround is not that good. You have to add a custom css with those properties:
``` css
.md-select-menu .md-select-menu-container {
  overflow: auto;
  max-height: 256px; //This size needs to be like that, for now
}
```

I tested this on selects with few options and worked. With a lot of options, like the combo of fonts, the scrollbar was applied. This is not gonna happen on the new md-select.

Thank you!

**Comment by cheresier** [comment]
Awesome, thanks again!

**Comment by marcosmoura** [comment]
Already implemented and will be shipped on the very next release (v0.3.0)
